Micro-dep-acct-eft-purge/readme.txt

```diff
@@ -0,0 +1,15 @@
+PowerOn Name:       BANNO.MICRO.BATCH.FM.V1.POW
```

Comment:
Not sure why the Letter file N/A is necessary. Can this readme header be consistent with the readme above? either both have the Letterfile N/A or not?

Reply:
Removed from both.